pkg/converter/drive_downgrade.go

```diff
@@ -75,7 +75,7 @@ func driveDowngradeV1Beta1ToV1alpha1(unstructured *unstructured.Unstructured) er
 		return err
 	}
 
-	klog.V(10).Infof("Converting directcsidrive: %v to v1alpha1", v1beta1DirectCSIDrive.Name)
+	klog.V(10).Infof("Converting directpvdrive: %v to v1alpha1", v1beta1DirectCSIDrive.Name)
```

Comment:
we have not yet moved to directpvdrive crd. we can call it `directcsidrive` only.